CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 Reactist follows [semantic versioning](https://semver.org/) and doesn't introduce breaking changes (API-wise) in minor or patch releases. However, the appearance of a component might change in a minor or patch release so keep an eye on redesigns and make sure your app still looks and feels like you expect it.
 
+## 2.0.3
+
+- [Tweak] Changed namespacing from `.reactist.popover {}` to `.reactist_popover {}`.
+
```

Comment:
This is not a minor change. This is actually a breaking change. All apps using reactist and customizing some styles will need to update their customizations.

Reply:
Updated the version to 3.0.0 and waiting for @henningmu's review.